### DISCUSSION TITLE
Any way to see the final nginx configuration

### DISCUSSION THREAD

**Discussion by tarunmittal24** [Kong/kubernetes-ingress-controller#6365] [Q&A]
Hi team,

Just curious to know, is there any way by which we can see the final nginx configuration applied in the pod, means after all the plugin instructions and other instructions, like we see in the normal nginx file, like file having the definition of the host, proxy_pass rewrite, etc.

**Comment by tao12345666333** [comment]
You can refer to this documentation to dump generated kong configurations. 

https://docs.konghq.com/kubernetes-ingress-controller/latest/reference/troubleshooting/#dumping-generated-kong-configuration

**Reply by tarunmittal24** [reply]
Thanks for this @tao12345666333 , this is helpful in debugging some issues, but this is not what i am looking for.

Looking for a nginx configuration, as i need to debug some stuff, where my plugin configuration is working in one pod and not in other.

https://github.com/Kong/kong/discussions/13422.

i got the solution for this, but need to confirm that, it is not applying on one pod where it is failing and get applied in other where it is not.
